deploy/AzureAppServices/Datadog.Trace.AzureAppServices.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Datadog.Trace.AzureAppServices</id>
+    <version>1.10.0</version>
+    <title>Datadog APM - .NET Tracer</title>
```

Comment:
Other products show up first in the list of extensions because they start with `.NET`. Maybe we should rename this to `.NET Tracer for Datadog APM`  😄